packages/auth/src/platform_browser/popup_redirect.test.ts

```diff
@@ -125,6 +126,49 @@ describe('platform_browser/popup_redirect', () => {
       );
     });
 
+    it('includes the App Check token in the url fragment if present', async () => {
+      await resolver._initialize(auth);
+      sinon
+        .stub(FAKE_APP_CHECK_CONTROLLER, 'getToken')
+        .returns(Promise.resolve({ token: 'fake-token' }));
+
+      await resolver._openPopup(auth, provider, event);
+
+      const matches = (popupUrl as string).match(/.*?#(.*)/);
+      expect(matches).not.to.be.null;
+      const fragment = matches![1];
+      expect(fragment).to.include('fac=fake-token');
+    });
+
+    it('does not add the App Check token in the url fragment if none returned', async () => {
+      await resolver._initialize(auth);
+      // Redundant, already set in mock_auth.ts but adding here for clarity
+      sinon
+        .stub(FAKE_APP_CHECK_CONTROLLER, 'getToken')
+        .returns(Promise.resolve({ token: '' }));
+
+      await resolver._openPopup(auth, provider, event);
+
+      const matches = (popupUrl as string).match(/.*?#(.*)/);
+      // The '#' character will not be included when the url fragment is not attached,
+      // so the url will not match the pattern
+      expect(matches).to.be.null;
+    });
+
+    it('does not add the App Check token in the url fragment if controller unavailable', async () => {
```

Comment:
Have you run `yarn lint:fix` ? (Or maybe it's `yarn format`?) to auto-wrap this

Reply:
Hmm, the linter isn't marking this as an issue (neither prettify nor `yarn lint:fix`)? I'll leave it as is for now